cmds/tui.c

```diff
@@ -142,6 +142,7 @@ static const char *help[] = {
 	"g             Show call graph for this function",
 	"R             Show uftrace report",
 	"r             Show uftrace report for this function",
+	"s             Show uftrace sorted report by KEYs",
```

Comment:
What about this?
```suggestion
	"s             Show uftrace report sorted by KEYs",
```

Reply:
Hmm.. it's not changing the view.  I'd rather say "Sort by the next column (in uftrace report)".